player/loadfile.c

```diff
@@ -259,33 +259,32 @@ static void print_stream(struct MPContext *mpctx, struct track *t)
     }
     char b[2048] = {0};
 
-    bool tracks_have_lang = false;
+    int max_lang_length = 0;
     for (int n = 0; n < mpctx->num_tracks; n++) {
-        if (mpctx->tracks[n]->lang) {
-            tracks_have_lang = true;
-            break;
-        }
+        if (mpctx->tracks[n]->lang)
+            max_lang_length = MPMAX(strlen(mpctx->tracks[n]->lang), max_lang_length);
     }
 
-    if (!isatty(STDOUT_FILENO)) {
-        APPEND(b, "%s ", t->selected ? BLACK_CIRCLE : WHITE_CIRCLE);
-    } else if (!t->selected) {
-        APPEND(b, "%s", TERM_ESC_GREY);
-    }
-    APPEND(b, "%-5s --%s=%-2d", tname, selopt, t->user_tid);
+    // Indent tracks after the Playing: line if present.
+    if (mpctx->playlist->num_entries > 1 || mpctx->playing->playlist_path)
+        APPEND(b, " ");
+    APPEND(b, "%s %-5s --%s=%-2d", t->selected ? BLACK_CIRCLE : WHITE_CIRCLE,
+           tname, selopt, t->user_tid);
     if (t->lang) {
-        APPEND(b, " --%s=%-7s", langopt, t->lang);
-    } else if (tracks_have_lang) {
-        FILL(b, 16);
+        char format[20];
+        snprintf(format, sizeof(format), " --%%s=%%-%ds", max_lang_length);
+        APPEND(b, format, langopt, t->lang);
+    } else if (max_lang_length) {
+        FILL(b, 9 + max_lang_length);
```

Comment:
```suggestion
        FILL(b, strlen(" --alang=") + max_lang_length);
```
to be more readable, what this 9 is.

Reply:
Done